Add documentation for Code Security configuration file (#35307)
DataDog/documentation: content/en/security/code_security/static_analysis/github_actions.md

By default, [Datadog Static Analyzer][8] detects the languages of your codebase and uses the default rulesets to analyze
your codebase.

> To specify and customize the rulesets, add a `code-security.datadog.yaml` file to your repository's root directory to define which rulesets to use.

Reviewer: First phrase is redundant with "to define which rulesets to use"

```suggestion
Add a `code-security.datadog.yaml` file to your repository's root directory to define which rulesets to use. For example:
```
